include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -847,6 +848,13 @@ def AIE_DMABDOp: AIE_Op<"dma_bd", [
 
     All strides are expressed in multiples of the element width (just like `len` and `offset`)
     **with the caveat that the inner-most dimension's stride must be 1**.
+
+    ## DMA zero padding on AIE-ML Devices
+
+    AIE-ML devices can apply zero padding at the buffer descriptor level, described with pairs of zero count 
+    before and after every dimension, up to three dimensions (four dimensions on memtiles). The zero-padding 
+    counts can be supplied to the `dma_bd` through an optional argument, an array of "tuple-like" attributes 
+    `bd_pad_layout<zero_pad_before, zero_pad_after>`.
```

Comment:
presumably this needs a verifier...
1) what happens if the device doesn't support it?
2) what's the range of valid values?
3) what happens if too many pads are specified?

Reply:
Verifier added to check for cases above.